metadata.yaml

```diff
@@ -1,6 +1,8 @@
 name: nginx
+maintainers: ['Adam Stokes <adam.stokes@ubuntu.com>']
```

Comment:
This is a layer and not a charm. Maintainers field should not be populated here. Otherwise it shows up for top layers including this.

charm proof is run against a compiled charm and not against a single layer